__tests__/webpack-relative-resolver.test.js

```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const AddonConfigurationRegistry = require('../addon-registry');
+const AddonConfigurationRegistry = require('../packages/registry/addon-registry');
```

Comment:
this should not depend on this current repository layout, rather use the package name, if possible.